gator CLI unit test yaml_output reports error for unexpected "evaluationmeta: null"

**What steps did you take and what happened:**

There is an issue with a unit test for the `gator` CLI.  if you run `go test -timeout 30s -run ^Test_formatOutput$ github.com/open-policy-agent/gatekeeper/cmd/gator/test -v` you get the following error reported.

```
=== RUN   Test_formatOutput/yaml_output
    /home/stevenlinde/src/github.com/linde/gatekeeper/cmd/gator/test/gatortest_test.go:94:   (
          	"""
          	... // 6 identical lines
          	            kind: kind
          	    enforcementaction: ""
        + 	    evaluationmeta: null
          	  violatingObject:
          	    bar: xyz
          	... // 2 identical lines
          	"""
          )

=== RUN   Test_formatOutput/json_output
--- FAIL: Test_formatOutput (0.00s)
    --- PASS: Test_formatOutput/default_output (0.00s)
    --- FAIL: Test_formatOutput/yaml_output (0.00s)
    --- PASS: Test_formatOutput/json_output (0.00s)
FAIL
FAIL	github.com/open-policy-agent/gatekeeper/cmd/gator/test	0.029s
```


**What did you expect to happen:**

i expected it to pass.

**Anything else you would like to add:**

this is a simple change to make it pass, assuming this is expected now to be in the output:

```
diff --git a/cmd/gator/test/gatortest_test.go b/cmd/gator/test/gatortest_test.go
index 102deef8..d56b4531 100644
--- a/cmd/gator/test/gatortest_test.go
+++ b/cmd/gator/test/gatortest_test.go
@@ -60,6 +60,7 @@ func Test_formatOutput(t *testing.T) {
         object:
             kind: kind
     enforcementaction: ""
+    evaluationmeta: null
   violatingObject:
     bar: xyz
   trace: xyz
```

**Environment:**

- Gatekeeper version:

as of `commit bed335f5c220e962922c887631a497f5bebe89ae (HEAD -> master, origin/master)`

- Kubernetes version: (use `kubectl version`): n/a

--- a/Makefile
+++ b/Makefile
@@ -101,7 +101,7 @@ all: lint test manager
 
 # Run tests
 native-test:
-	GO111MODULE=on go test -mod vendor ./pkg/... ./apis/... -race -bench . -coverprofile cover.out
+	GO111MODULE=on go test -mod vendor ./pkg/... ./apis/... ./cmd/gator/... -race -bench . -coverprofile cover.out
 
 # Hook to run docker tests
 .PHONY: test

--- a/cmd/gator/test/gatortest_test.go
+++ b/cmd/gator/test/gatortest_test.go
@@ -60,6 +60,7 @@ func Test_formatOutput(t *testing.T) {
         object:
             kind: kind
     enforcementaction: ""
+    evaluationmeta: null
   violatingObject:
     bar: xyz
   trace: xyz

